AnnoyancesFilter/sections/cookies_specific.txt

```diff
@@ -5636,7 +5636,7 @@ arpaceramiche.it,firefox-usb.com,m64.info##.jb.cookie
 softpedia.com##.jobnotice_cnt
 yahoo.com##.js-applet.windowshade .fixed-win.Px-10.Bd-1
 chessable.com,leroymerlin.fr##.js-banner-cookie
-daimler-tss.com,gofundme.com##.js-cookie-banner
+made.com,##.cookiedaimler-tss.com,gofundme.com##.js-cookie-banner
```

Comment:
`made.com,daimler-tss.com,gofundme.com##.js-cookie-banner`

Reply:
Oh I didn't see this, I will fix this